recipes/moltenvk/all/patches/0002-allow-c++11-std-1.1.5.patch

```diff
@@ -0,0 +1,16 @@
+upstream issue: https://github.com/KhronosGroup/MoltenVK/issues/1474
+
+--- a/MoltenVK/MoltenVK/Commands/MVKCommandBuffer.mm
++++ b/MoltenVK/MoltenVK/Commands/MVKCommandBuffer.mm
+@@ -769,7 +769,10 @@ void MVKCommandEncoder::markTimestamp(MVKTimestampQueryPool* pQueryPool, uint32_
+ 	if (sampPts) {
+ 		for (uint32_t qOfst = 0; qOfst < queryCount; qOfst++) {
+ 			if (mvkIsAnyFlagEnabled(sampPts, MVK_COUNTER_SAMPLING_AT_PIPELINE_STAGE)) {
+-				_timestampStageCounterQueries.push_back({ pQueryPool, query + qOfst });
```

Comment:
If c++11 support is that important, perhaps you can do `emplace_back(pQueryPool, query + qOfst)`.
And eventually send it upstream?

Reply:
it is not that important, maybe I'll drop these patches and require C++14 or C++17 since upstream wants to move to C++17.